packages/dds/tree/src/feature-libraries/editable-tree/editableTree.ts

```diff
@@ -76,15 +85,47 @@ export interface EditableTree {
     readonly [proxyTargetSymbol]: object;
 
     /**
-     * Anchor to this node.
-     * Valid as long as this EditableTree's context is not freed.
-     * Might not point to any node if this node is deleted from the document.
-     *
-     * TODO: When a proper editing API is exposed on EditableTree directly,
-     * this should become an implementation detail and rbe removed from this API surface.
+     * Creates a child of this node.
+     */
+    readonly [insertNodeSymbol]: (key: string, value: ITreeCursor) => boolean;
+
+    /**
+     * Sets value of a child of this node.
+     */
+    readonly [setValueSymbol]: (key: string, value: unknown, typeName: TreeSchemaIdentifier) => boolean;
```

Comment:
Instead of adding a way to edit the value of children, I recommend implementing the TODO already documented on editable tree, and provide an API to allow getting children without unwrapping them (we need that for more than just value editing (ex: getting anchors to them), so we might as well use it to solve this problem as well)

Then you can have a way (here) to set the value of the current node.

This avoids most of the issue I mentioned above, and also makes it possible to set values on root node without having to have a parent for them (which is impossible for the root).